src/lsst/cmservice/common/butler.py

```diff
@@ -31,6 +31,8 @@ def remove_run_collections(
         raise errors.CMNoButlerError(e) from e  # pragma: no cover
     try:  # pragma: no cover
         butler.registry.removeCollection(collection_name)
+    except MissingCollectionError:
+        pass
```

Comment:
Is this something we'd like be noted in the server log when it happens, or not really?

Reply:
Not really.  It just means that the bps submit didn't actually get to the point of making the output collection...